tests/testthat/example/out.R

```diff
@@ -115,11 +115,11 @@ test <- function() {
 
   nested(function_call(
   "in",
- multiple, lines))
+      multiple, lines))
 
   nested(
   function_call(with),
- many
+      many
```

Comment:
I think it's fine, because the rule `add_space_after_comma()` was not checking for line break (which we need if we want to support raw indention) and this is the flat styling anyways. The token is now where it is in the in.R file.